frontend/src/Components/Visualisations/TextComponent.js

```diff
@@ -0,0 +1,7 @@
+import React from "react";
+
+const TextTable = (props) => {
+  return <p>hello</p>;
```

Comment:
This component should take the content of text as parameter and render it. As it is not yet requested, it should not be included as part of current PR.